1214-update-community-roadmap-page (#1220)
mi6/ic-design-system: src/content/structured/community/roadmap.mdx

- Canary release of [date input component](/components/date-input).
- Canary release of [date picker component](/components/date-picker).
- Canary release of [multi-select component](/components/multi-select).

> - Canary release of [toggle button component](/components/toggle-button).

Reviewer: ```suggestion
- Release of [toggle button component](/components/toggle-button).
```

> - Canary release of [accordion component](/components/accordion).

Reviewer: ```suggestion
- Release of [accordion component](/components/accordion).
```

- Release of [toggle button component](/components/toggle-button).
- Release of [accordion component](/components/accordion).
- Improvements to code previews (StackBlitz, hide/show code, imports).
- Refresh of website including homepage and developer and designer landing pages.
- Improvements to testing (visual regression, Cypress integration).

> - Custom theme for AG grid component.

Reviewer 2: ```suggestion
- Custom theme for AG Grid component.
```

> - Token and theming of code components for dark mode.

Reviewer 2: I think it would be fine to just put "components"? Not sure "code components" is very clear
```suggestion
- Tokens and theming of components for dark mode.
```

## Current focus

- Canary release of drawer component.
- Explore theming and customisation of components.
- Design and build clickable chip component.
- Design and build skip to content link component.
- Design and build selectable card component.

> - Mature and move existing canary components to beta.

Reviewer 2: ```suggestion
- Mature and move existing canary components to Beta.
```

- Add guidance on AI patterns, dark mode and colour tokens.

> - Token and theming of Figma components in preparation for dark mode.

Reviewer 2: ```suggestion
- Tokens and theming of Figma components in preparation for dark mode.
```
